src/client/Microsoft.Identity.Client/Platforms/netcore/NetCoreHttpClientFactory.cs

```diff
@@ -9,32 +10,30 @@ namespace Microsoft.Identity.Client.Platforms.netcore
 {
     internal class NetCoreHttpClientFactory : IMsalHttpClientFactory
     {
-        private static HttpClient s_httpClient;
```

Comment:
For customers using this library before this fix, in scenarios where there are multiple instances of this, they will get "Properties can only be modified before sending the first request" HttpClient exception, that should be documented @trwalke